src/codecov_ats.sh

```diff
@@ -114,31 +114,31 @@ if [[ -z $response && -n $INPUTS_OVERRIDE_BASE_COMMIT ]]; then
 	exit 1
 fi
 
-response=$(echo $response | sed 's/,//g')
-runner_options=$(echo $response | sed 's/^.*runner_options\": \[//' | sed 's/\].*$//')
-ats_tests_to_run=$(echo $response | sed 's/^.*ats_tests_to_run\": \[//' | sed 's/\].*$//')
-ats_tests_to_skip=$(echo $response | sed 's/^.*ats_tests_to_skip\": \[//' | sed 's/\].*$//')
-
-if [[ -z $runner_options ]]; then
-    say "$y==>$x Could not find 'runner_options', defaulting to '--cov-context=test'"
-    runner_options="--cov-context=test"
-fi
+# Post process label-analysis response
 
-if [[ -z $ats_tests_to_run ]]; then
-    say "$y==>$x No tests to run, picking random test"
-    ats_tests_to_skip_array=($ats_tests_to_skip)
 
-    if [[ -z $ats_tests_to_skip ]]; then
-        say "$y==>$x No tests to skip, running all tests"
```

Comment:
There are the 2 cases here that I'd like to make sure we're doing.

1. If no tests are selected to run, pick one at random (so that we dont' run all tests)
2. If there are no tests to skip, don't pass in any tests so that it runs all tests

Reply:
Point 1 I understand, but can you clarify point 2?
You mean "don't pass any tests" to the tests to run?

Reply:
I wouldn't be to eager to do it if that's the case.

First because changing the action output is exactly to support this scenario, so it's unnecessary to treat this case differently.
Second because, if you think about it, there's a chance that not passing any info would actually cause us to tell the user to run _more_ tests than passing them all, if the user configures [collect_test_options](https://docs.codecov.com/docs/label-analysis#configuration). I do think this is unlikely to happen, but still possible.

Reply:
Oh, that's a good point. I hadn't considered the `collect_test_options`